packages/scripts/README.md

```diff
@@ -0,0 +1,3 @@
+# Glint scripts
+
+A handful of tools for working with Glint—e.g. for migrations.
```

Comment:
TODO: I need to add an explanation of the `migrate-glintrc` script, minimally how to use it.